perf/smallbank_workload/src/smallbank_tranformer.rs

```diff
@@ -28,20 +29,66 @@ use sawtooth_sdk::signing;
 
 use playlist::bytes_to_hex_str;
 use playlist::make_addresses;
-use smallbank::SmallbankTransactionPayload;
+use smallbank::{SmallbankTransactionPayload, SmallbankTransactionPayload_PayloadType};
 
 /// Transforms SmallbankTransactionPayloads into Sawtooth Transactions.
 pub struct SBPayloadTransformer<'a> {
     signer: &'a signing::Signer<'a>,
+    signature_by_customer_id: HashMap<u32, String>,
 }
 
 impl<'a> SBPayloadTransformer<'a> {
     pub fn new(signer: &'a signing::Signer) -> Self {
-        SBPayloadTransformer { signer: signer }
+        SBPayloadTransformer {
+            signer: signer,
+            signature_by_customer_id: HashMap::new(),
+        }
+    }
+
+    fn add_signature_if_create_account(
+        &mut self,
+        payload: &SmallbankTransactionPayload,
+        signature: String,
+    ) {
```

Comment:
It feels like this function is too specific/knows too much about the types being passed. I think something like `track_customer_signature(customer, signature)` would be clearer, and then the function using this can handle checking whether it is a create account and getting the customer id.